src/lib/encoder.ts

```diff
@@ -143,6 +143,123 @@ const decodeCompactBytesArray = (compactBytesArray: string): string[] => {
   return encodedValues;
 };
 
+const encodeBytesNCompactBytesArray = (
+  values: string[],
+  numberOfBytes: number,
+): string => {
+  for (let i = 0; i < values.length; i++) {
+    if (values[i].length > numberOfBytes * 2 + 2)
+      throw new Error(
+        `Hex bytes${numberOfBytes} value at index: ${i} is using ${
+          (values[i].length - 2) / 2
+        } bytes, which exceedes ${numberOfBytes}`,
```

Comment:
Here it should be exceeds and not exceedes no ?

Reply:
Yep, good catch